src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenTupleTest.cs

```diff
@@ -554,7 +554,7 @@ static void Main()
 }
 " + trivial2uple + trivial3uple;
 
-            CreateCompilationWithMscorlib(source).VerifyDiagnostics(
+            CreateCompilationWithMscorlib(source, parseOptions: TestOptions.ExperimentalParseOptions).VerifyDiagnostics(
```

Comment:
I wasn't sure the recommended way to do this. Vlad suggested experimental options would be ok.
I'll add `TestOptions.ExperimentalTupleParseOptions` and use that instead.